File: 1 - Sequenciais/sequenciais6.c
//Faça um programa que peça o raio de um círculo, calcule e mostre sua área.

#include <stdio.h>
#include <math.h>

int main()
{
  float raio;
  double area;
  printf("Digite o raio da circunferência: \n");
  scanf("%f", &raio);
  area = M_PI * pow(raio, 2);
  printf("O valor da area eh de: %.2f", area);
  return 0;
}

File: 1 - Sequenciais/sequenciais6.php
<?php #Faça um programa que peça o raio de um círculo, calcule e mostre sua área. ?>

<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Sequenciais 6 - PHP</title>
  </head>
  <body>
    <?php if (!empty($_GET)): ?>
      <?= 'A área da circunferência é de: ' . ( pow($_GET['raio'], 2)  * M_PI); ?>
    <?php else: ?>
      <form action="sequenciais6.php" method="get">
        Digite o valor do raio de um círculo: <input type="text" name="raio"/>
        <input type="submit" value="Enter">
      </form>
    <?php endif; ?>
  </body>
</html>
